.github/actions/install/action.yml

```diff
@@ -0,0 +1,13 @@
+runs:
+  using: composite
+  steps:
+    - run: yarn versions
+      shell: bash
+    - uses: actions/cache@v2
+      id: cache
+      with:
+        path: node_modules
+        key: yarn-${{ runner.os }}-${{ hashFiles('package.json') }}
+    - run: yarn install
```

Comment:
This should just be `yarn`, shouldn't it?

Reply:
Both work, I can change it if you prefer the shorthand.

Reply:
Nah it's fine.